Lombiq.Tests.UI/Extensions/TenantsUITestContextExtensions.cs

```diff
@@ -49,7 +49,7 @@ public static class TenantsUITestContextExtensions
 
         if (!string.IsNullOrEmpty(featureProfile))
         {
-            await context.SetDropdownByTextAsync("FeatureProfile", featureProfile);
+            await context.ClickReliablyOnAsync(By.XPath($"//option[@value='{featureProfile}']"));
```

Comment:
Why can't `SetDropdownByTextAsync` be used?

Reply:
The editor has changed seemingly it is no longer a dropdown. It's a list where you have to click and choose the item.